lil_aretomo/aretomo.py

```diff
@@ -23,7 +24,8 @@ def run_aretomo_alignment(
         nominal_rotation_angle: Optional[float] = None,
         n_patches_xy: Optional[Tuple[int, int]] = (5, 4),
         thickness_for_alignment: Optional[float] = 1500,
-        correct_tilt_angle_offset: Optional[bool] = False	
+        correct_tilt_angle_offset: Optional[bool] = False,
+        gpu_id: Optional[str] = None	
```

Comment:
would prefer that our interface take a tuple of ints and convert -> don't want to impose RELION-isms on our Python API
```suggestion
        gpu_id: Optional[Tuple[int, ...]] = None	
```

What do you think?

Reply:
would need a few other changes to make this work ofc

Reply:
Sweet, will have tuple of ints as input and ask for rln to pass the info as a tuple of ints!

Reply:
test out what the CLI parser wants for tuple of ints before asking just to make sure :) I think it's just space separated ints `0 1 2 3`

Reply:
Typer won't accept an ellipsis as a type hint which is annoying, may have to keep it as a str?

Reply:
This is an issue in tomo_PP not lil_aretomo

Reply:
Could have: relion takes string (e.g. 0:1:2:3), tomo_PP takes this string and removes the ":", passes to lil_AT as a tuple. Does this make sense?

Reply:
cool okay, can you rename to gpu_ids and specify the expected formatting in the docstring? :)

Reply:
will open an issue at typer

Reply:
wait a sec, what if we type as `List[int]` - should work right?

Reply:
I'll give that a go!

Reply:
No, still doesn't like it. I can't pass a list on the CLI, only thing that works is List[int] which only allows me to pass one value. If I change to List[int,...] I get a slightly different error with:  TypeError: Parameters to generic types must be types. Got Ellipsis.